fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/importhandler/clientcupoincrement/ClientCupoIncrementImportHandler.java

```diff
@@ -207,7 +208,17 @@ private Count importEntity(final Workbook workbook, final List<ClientCupoIncreme
                 clientCupoIncrementSheet.getRow(clientCupoIncrementData.getRowIndex())
                         .createCell(ClientCupoIncrementConstants.PREVIOUS_MAXIMUM_CUPO_AMOUNT_COL)
                         .setCellValue(previousMaximumCupoAmount.doubleValue());
-                if (maximumCupoAmount.compareTo(previousMaximumCupoAmount) < 0) {
+
+                if (maximumCupoAmount.compareTo(previousMaximumCupoAmount) > 0) {
+                    errorCount++;
+                    errorMessage = "No puede modificarse ya que el nuevo cupo que sugiere es menor al actual";
+                    ImportHandlerUtils.writeErrorMessage(clientCupoIncrementSheet, clientCupoIncrementData.getRowIndex(), errorMessage,
+                            ClientCupoIncrementConstants.STATUS_COL);
+                    clientCupoIncrementSheet.setColumnWidth(ClientCupoIncrementConstants.STATUS_COL,
+                            TemplatePopulateImportConstants.EXTRALARGE_COL_SIZE);
+                    continue;
+                }
```

Comment:
@tabrez-fiter compreTo method returns one of the three outputs. -1, 0 or 1. This condition will never be true. You have already checked above if compareTo returns value greater than 0

Reply:
This is to compare between the input value cannot be more than the value of Cupo aprobado or the maximum increase in cupo that is allowed, so there are two criteria where cupo cannot be changed through this feature if the value of Cupo aprobado is 0 and cannot increase Cupo aprobado greater than the value of Cupo aprobado before

Reply:
Please see my comment above again. Line 221
`if (maximumCupoAmount.compareTo(previousMaximumCupoAmount) > 1) {`
This statement can never be true because compareTo returns only -1, 0 or 1. You have already checked if it returns value greater than 0 so this statement is redundant

Reply:
Line 128 is redundant also.